api/Document.json

```diff
@@ -7940,7 +7940,7 @@
               "version_added": "10.1"
             },
             "safari_ios": {
-              "version_added": "10.3"
+              "version_added": false
```

Comment:
This does not seem plausible, that the `onpointerlockchange` event handler attribute should be supported in Safari but not Safari iOS, while `element.requestPointerLock()` is claimed to be supported in both.

What were the tests results for each here?

Reply:
It appears that `api.Element.requestPointerLock` is not supported in Safari iOS, which lines up with this data.

Reply:
Great, thanks!